.github/gh_matrix_builder.py

```diff
@@ -224,6 +246,18 @@ def macos_config(overrides):
             }
         )
     )
+    m["include"].append(
+        build_debug_config(
+            {
+                "pg": 15,
+                "snapshot": "snapshot",
+                # below tests are tracked as part of #4972
+                "installcheck_args": "SKIPS='dist_move_chunk' "
+                # below tests are tracked as part of #4835
+                "IGNORES='telemetry_stats partialize_finalize'",
```

Comment:
Could you indent this line (or add a comment) to indicate that this belongs to `installcheck_args`?

Reply:
I have now defined a new common pg15_installcheck_args variable inside the script with proper indentation and comment. It is now used by all the PG15 builds.